app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\UserRequest;
+use App\User;
+use Exception;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Support\Facades\Hash;
+
+class UserController extends Controller
+{
+    public function users()
+    {
+        try {
+
+            return response()->json(User::all(), 200);
+        } catch (ModelNotFoundException $exception) {
+
+            return response()->json([
+                'message' => 'Users not found.'
+            ], 404);
+        } catch (Exception $exception) {
+
+            return response()->json([
+                'message' => 'Oops! Something went wrong.'
+            ], 500);
+        }
```

Comment:
Sobre os `return` statements, eles só precisam receber uma quebra de linha caso existam outras tratativas de código anteriores a ele, quando o `return` é o único código dentro do bloco não precisa quebrar uma linha =)

```suggestion
        } catch (Exception $exception) {
            return response()->json([
                'message' => 'Oops! Something went wrong.'
            ], 500);
        }
```